src/components/CarouselCard/__tests__/utils.spec.js

```diff
@@ -57,4 +58,38 @@ describe('<CarouselCard/> utils', () => {
             expect(getItemIndex(children, 'indicator-4')).toBe(-1);
         });
     });
+
+    describe('carouselCardContainerStyles', () => {
+        it('should return null if a ref is not passed', () => {
+            expect(carouselCardContainerStyles()).toBeNull();
+        });
+        it('should return null if ref passed does not have a parentNode', () => {
+            const ref = {};
+            expect(carouselCardContainerStyles(ref)).toBeNull();
+        });
+        it('should return an object with a heght of 100% if the parentNode the ref passed has an height set', () => {
```

Comment:
height

Reply:
the alt attribute is only for img element so here we need to use the AssistiveText component in order to screen readers take the assistive text. Then:
```
<div className="rainbow-carousel-image_content-image-container">
    <div className="rainbow-carousel-image_image" style={this.getImageSrc()} />
    <AssistiveText text={assistiveText} />
</div>
```